libs/agno/agno/db/schemas/skill.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import Any, Dict, List, Optional
+
+from agno.utils.dttm import now_epoch_s, to_epoch_s
+
+
+@dataclass
+class Skill:
+    """Model for Skills stored in database.
+
+    A skill provides structured instructions, reference documentation,
+    and optional scripts that an agent can access to perform specific tasks.
+
+    Attributes:
+        id: SHA256 content hash used as primary key
+        name: Unique skill name
+        description: Short description of what the skill does
+        instructions: Full SKILL.md body (the instructions/guidance for the agent)
+        metadata: Optional metadata (version, author, tags, etc.)
+        version: Integer version number for the skill
+        scripts: List of script objects with name and content
+        references: List of reference objects with name and content
+        created_at: Epoch timestamp when skill was created
+        updated_at: Epoch timestamp when skill was last updated
+    """
+
+    id: str
+    name: str
+    description: str
+    instructions: str
```

Comment:
now we'd also need something like source_type: local or db 
maybe source_path also for the file path of local directory.